modin/core/dataframe/algebra/default2pandas/groupby.py

```diff
@@ -205,7 +205,7 @@ def fn(
                 and by.name in df
                 and df[by.name].equals(by)
             ):
-                by = by.name
+                by = [by.name]
```

Comment:
[the following code](https://github.com/dchigarev/modin/blob/330d1733c1ca2c12b13b80c514b628acc4bc700f/modin/core/dataframe/algebra/groupby.py#L227) treats `by` as a list-like sequence, so wrapping the name into a list